src/components/Camera/index.js

```diff
@@ -36,8 +37,11 @@ const CameraPure = ({
 }: Props) => (
   <div className={classNames(style.camera, className)}>
     {renderTitle}
-    <div className={classNames(style.container, containerClassName)}>
-      <div className={style.webcamContainer} aria-label={translate('accessibility.selfie_camera_view')}>
+    <div
+      className={classNames(style.container, containerClassName)}
+      aria-label={translate('accessibility.camera_view')}
+      tabIndex='0'>
```

Comment:
I would be inclined to have this as a "group" role and use `aria-describedby` because this is not a focusable element. This will make the element accessible both to keyboard and to screen readers without making the element focusable.

Reply:
Ah yes. I forgot that the view shouldn't be accessible with keyboard tab.